Game/Resources/Constant Scene/Minecart/Script/Controller.as

```diff
@@ -23,7 +23,10 @@ class Controller {
         //if (self.name == "Right Controller") {
         //    print("Controller position: (" + self.GetWorldPosition().x + ", " + self.GetWorldPosition().y + ", " + self.GetWorldPosition().z + ")\n");
         //}
-            
+        if(!IsVRActive()){
```

Comment:
Is this used for anything?

Reply:
So the lantern that is connected to the camera wont spawn in fuck knows where